### PR TITLE
Create the atlas files from wago.tools automatically

### PR DESCRIPTION
**Pull request by InfusOnWoW**
Fixes: #4776

### CODE REVIEW

**Comment by InfusOnWoW** [comment]
The result of the action can be seen at https://github.com/InfusOnWoW/WeakAuras2/pull/2/files

Checking out [update-atlasfiles](https://github.com/InfusOnWoW/WeakAuras2/tree/update-atlasfiles) allows testing the generated atlas file list. 

I did spot checks of the difference between what this generates and what the old method of naming the atlas files generated, and indeed #4776 is correctly desribing a more accurate algorithm.

**Review by mrbuds — APPROVED** [review]

File: .github/workflows/atlas-update.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+      - name: Update Atlas files from wago.tools
+        run: |
+          cd .github/scripts/
+          ./wagotools-atlas.sh
+        shell: bash
+
+      - name: Cleanup luarocks
+        run: |
+             rm -rf .luarocks
```
Comment (Stanzilla) [inline .github/workflows/atlas-update.yml:30]: why is that needed?

Comment (Stanzilla) [reply .github/workflows/atlas-update.yml:30]: Ah fair, could have put it in .gitignore but doesn't really matter
